ClemBot.Bot/bot/cogs/translate_cog.py

```diff
@@ -86,22 +24,38 @@ def __init__(self, bot):
 
     @ext.group(case_insensitive=True, invoke_without_command=True)
     @ext.long_help(
-        'Allows you to translate words or sentences by either specifying both the input and output language with the text to translate, or just the output language and the text to translate. run \'translate languages\' to see available languages')
+        'Allows you to translate words or sentences by specifying an output language and autodetecting the language you entered. Use !translate m for specifying the language you entered. Use \'translate languages\' to see available languages')
```

Comment:
Break this up into multiple strings on new lines, python will concatencate them automatically